source/opt/decoration_manager.cpp

```diff
@@ -300,6 +275,44 @@ void DecorationManager::CloneDecorations(uint32_t from, uint32_t to,
   }
 }
 
+void DecorationManager::RemoveDecoration(ir::Instruction* inst) {
+  switch (inst->opcode()) {
+    case SpvOpDecorate:
+    case SpvOpDecorateId:
+    case SpvOpMemberDecorate: {
```

Comment:
If we handle SpvOpName in AddDecoration, it should also be handled here.

Reply:
I'll remove the OpName from AddDecoration.